server/chat-commands.js

```diff
@@ -3740,6 +3740,39 @@ const commands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ & ~`],
 
+	acceptdraw: 'acceptdraw',
+	offerdraw(target, room, user, connection, cmd) {
+		const battle = room.battle;
```

Comment:
Move these two variables down 5 lines to where theyre actually used

Reply:
`battle` is used exactly one line down, so just moved `playerIDs` (now `playerIds`).